CHANGELOG.md

```diff
@@ -79,6 +102,35 @@
   clearly labeled `experimental`.  Regardless, we apologize if this causes you
   inconvenience or additional work.
 
+**OTHER CHANGES**:
+
+* fix(storage): eliminate dangling function references ([#7366](https://github.com/googleapis/google-cloud-cpp/pull/7366))
+* doc: add gRPC steps to storage quickstart ([#7358](https://github.com/googleapis/google-cloud-cpp/pull/7358))
+* fix(storage): restore deprecation warnings on MSVC ([#7325](https://github.com/googleapis/google-cloud-cpp/pull/7325))
+* fix(storage): build with MSVC+x86 ([#7323](https://github.com/googleapis/google-cloud-cpp/pull/7323))
+* feat(GCS+gRPC): reduce data copying in downloads ([#7303](https://github.com/googleapis/google-cloud-cpp/pull/7303))
+* feat(GCS+gRPC): support timeouts for all requests ([#7299](https://github.com/googleapis/google-cloud-cpp/pull/7299))
+* feat(storage): support timeouts for all requests ([#7295](https://github.com/googleapis/google-cloud-cpp/pull/7295))
+* feat(storage): use generic parameters when resuming uploads ([#7292](https://github.com/googleapis/google-cloud-cpp/pull/7292))
+* feat(storage): support base URLs with existing query parameters ([#7291](https://github.com/googleapis/google-cloud-cpp/pull/7291))
```

Comment:
I think this is an internal-only change, I should have labeled somehting else.

Reply:
Removed.